Rakefile

```diff
@@ -1,6 +1,16 @@
+begin
+  require 'bundler/setup'
+rescue LoadError
+  puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
+end
+
 require "bundler/gem_tasks"
 require 'rspec/core/rake_task'
 
+load 'rails/tasks/engine.rake'
```

Comment:
I don't think you need/want these tasks without a checked in test app (although maybe engine_cart ought to pull them in..)